[Feat] PointView 섹션 0,1,2,3 세팅 #30

jaminleee opened the pull request:
## 🔥*Pull requests*

⛳️ **작업한 브랜치**
- #30

👷 **작업한 내용**
<!-- 작업한 내용을 적어주세요. -->

## 🚨 참고 사항
<!-- 참고할 사항이 있다면 적어주세요. -->

## 📸 스크린샷
|기능|스크린샷|
|:--:|:--:|
|GIF|<img src = "" width ="250">|

## 📟 관련 이슈
- Resolved: #30

jaminleee committed:
[Fix] conflict 해결 (#30)

# Conflicts:
#	NaverPay.xcodeproj/project.pbxproj
#	NaverPay/Scenes/Place/ViewControllers/PlaceViewController.swift

jaminleee committed:
[Feat] PointViewController, cell 생성 및 이미지 파일 추가

jaminleee committed:
[Feat] PointView 섹션 1,2,3,4 레이아웃 설정

jaminleee committed:
[Feat] alertBanner 이미지 추가

jaminleee committed:
[Feat] GraphCollectionViewCell 배너 이미지뷰 추가

jaminleee committed:
[Fix] Pull main, conflict 해결

# Conflicts:
#	NaverPay.xcodeproj/project.pbxproj

jaminleee committed:
[Feat] PointView 섹션 0 UI 구현

jaminleee committed:
[Feat] 이미지 파일 등 Extension 수정

jaminleee committed:
[Feat] PointViewController 세팅

jaminleee committed:
[Feat] GraphCollectionViewCell UI 구현

jaminleee committed:
[Feat] GraphCollectionViewCell 레이아웃 구현 (#30)

sjk4618 commented on NaverPay/Scenes/Point/Cells/BannerCollectionViewCell.swift:
```diff
@@ -0,0 +1,43 @@
+
+import UIKit
+
+class BannerCollectionViewCell: UICollectionViewCell {
```
final~~

sjk4618 commented on NaverPay/Scenes/Point/Cells/GraphCollectionViewCell.swift:
```diff
@@ -0,0 +1,259 @@
+    
+    private let dateLabel = NPLabel(font: .font(.detail_smbold_12), color: .grayscale_gray6)
+    
+    private let category1Button: UIButton = {
```
숫자넣기보단 First 이런식이면 더 좋을 것 같아요~

sjk4618 commented on NaverPay/Scenes/Point/Cells/GraphCollectionViewCell.swift:
```diff
@@ -0,0 +1,259 @@
+    }
+    
+    private func setLayout() {
+        contentView.backgroundColor = .bg_white
+        monthLabel.text = "11월"
+        descLabel.text = "이달에는 결제수단 혜택을 많이 받았어요"
+        totalLabel.text = "총 적립 혜택"
+        amountLabel.text = "11,500원"
+        dateLabel.text = "2023.11.01. \n~ 11.30."
+        dateLabel.numberOfLines = 2
+        dateLabel.textAlignment = .center
```
요런거는  따로 메서드 빼주면 좋을 것 같네요~

sjk4618 reviewed (commented):
수고하셨습니다!

kim-seonwoo reviewed (commented):
수고하셨습니다~~!!

jaminleee committed:
[Chore] 코드 리뷰 반영

jaminleee committed:
[Fix] Conflict 해결

# Conflicts:
#	NaverPay.xcodeproj/project.pbxproj
#	NaverPay/Application/SceneDelegate.swift
#	NaverPay/Scenes/Benefit/ViewControllers/BenefitViewController.swift
